contracts/governance/locking/LockingRelock.sol

```
@@ -117,6 +117,7 @@ abstract contract LockingRelock is LockingBase {
    */
   function rebalance(uint256 id, address account, uint96 bias, uint96 residue, uint96 newAmount) internal {
     require(residue <= newAmount, "Impossible to relock: less amount, then now is");
+    require(newAmount >= 1e18, "amount is less than minimum");
```

Why did you move this check into the rebalance function? I would say it better fits into the verification() function that is called before rebalance() and already performs a check on newAmount 🤔 
We could also think about adding the check on top of relock() to fail early and be more consistent with lock().

---

Makes sense, will shuffle around 👌🏼